pkg/sdn/plugin/ovscontroller.go

```diff
@@ -207,12 +210,34 @@ func (oc *ovsController) cleanupPodFlows(podIP string) error {
 	return otx.EndTransaction()
 }
 
-func (oc *ovsController) SetUpPod(hostVeth, podIP, podMAC string, vnid uint32) (int, error) {
+func getPodNote(containerID string) (string, error) {
+	bytes, err := hex.DecodeString(containerID)
+	if err != nil {
+		return "", fmt.Errorf("failed to decode container ID %q: %v", containerID, err)
+	}
+	if len(bytes) != 32 {
+		return "", fmt.Errorf("invalid container ID %q length; expected 16 bytes", containerID)
+	}
+	var note string
+	for _, b := range bytes {
+		if len(note) > 0 {
+			note += "."
+		}
+		note += fmt.Sprintf("%02x", b)
+	}
+	return note, nil
+}
```

Comment:
So this is equivalent to just adding a "." after every 2nd character in containerID, right? Except that it fails if containerID isn't actually a hex string.